### DISCUSSION TITLE
Possible to check whether Pixel Refresher is running/has run?

### DISCUSSION THREAD

**Discussion by dekiesel** [supersaiyanmode/PyWebOSTV#70] [Q&A]
Hi,

Is it possible to check whether the pixel refresher is still running or when it has been run? The pixel refresher is (afaik) the program to prevent burn-in.

**Comment by supersaiyanmode** [comment]
Hi, I am not aware of any APIs that is exposed by the TV that gives us this information, sorry. If you have come across any pointers/documentation/etc around this since you asked this question, please let me know -- I'll try to evaluate if this library can provide it.
